docs/references/expo/expo-impersonation.mdx

```diff
@@ -0,0 +1,508 @@
+---
+description: Learn how to build custom Impersonation into your Expo application
+---
+
+# Impersonation in Expo
+
+> [!WARNING]
+> The following flow is built assuming your Clerk app is using the default instance settings. If you have changed your instance settings, you may need to make changes to the examples.
+
+[User Impersonation](/docs/custom-flows/user-impersonation#impersonate-a-user-from-the-clerk-dashboard) is a way to offer customer support by logging into your applications from their accounts. Doing so enables you to directly reproduce and remedy any issues they're experiencing. In this example, you'll learn how to build Clerk's user impersonation into your Expo application.
+
+## Before you start
+
+1. Follow [Clerk's Expo quickstart](/docs/quickstarts/expo) to ensure your project is configured properly.
+1. For simplifying the code example and getting started with development, this is using the new, experimental Expo API Routes. Please review https://docs.expo.dev/router/reference/api-routes/ for more information. You can use your own API server instead of these Expo API routes.
+1. You must configure your Clerk instance to enable **Email and Password** authentication:
+    1. Navigate to the Clerk Dashboard in the navigation sidebar, go to **User & Authentication > [Email, Phone, Username](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username)**.
+    1. Ensure that **Email address** is required, and **Username** is not required.
+    1. Ensure that **Password** is required.
+1. Install the following dependencies:
+    <CodeBlockTabs options={["npm", "yarn", "pnpm"]}>
+    ```bash {{ filename: "terminal" }}
+    npm i expo-linking && npm i @clerk/types
+    ```
+
+    ```bash {{ filename: "terminal" }}
+    yarn add expo-linking && yarn add @clerk/types
+    ```
+
+    ```bash {{ filename: "terminal" }}
+    pnpm add expo-linking && pnpm add @clerk/types
+    ```
+    </CodeBlockTabs>
+1. Now that Clerk is configured, you'll need to [deploy to Vercel](https://docs.expo.dev/router/reference/api-routes/#vercel).
+
+<Steps>
+
+### Create a `(dashboard)` route
+
+> [!WARNING]
```

Comment:
This generally applies to all custom flows, but I also don't think its helpful. Customers understand this and are editing before production.